src/classes/modules/java.base/java/util/Calendar.java

```diff
@@ -0,0 +1,3279 @@
+package java.util;
+
+import java.io.IOException;
```

Comment:
Double-check which imports are actually needed, remove unused ones.

Reply:
Calendar.java class has been cleaned up to contain only the functionality that is strictly necessary for DateFormat to work.